pyproject.toml

```diff
@@ -0,0 +1,35 @@
+[build-system]
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "xcube_clms"
+dynamic = ["version"]
+authors = [
+  {name = "Yogesh Kumar Baljeet Singh", email = "yogesh.baljeetsingh@brockmann-consult.de"}
+]
+description = """\
+  xcube datastore for accessing CLMS datastore\
+  """
+keywords = [
+  "analysis ready data", "data science", "datacube", "xarray", "zarr", "xcube", "clms"
+]
+readme = {file = "README.md", content-type = "text/markdown"}
+license = {text = "MIT"}
+requires-python = ">=3.8"
+dependencies = [
+]
+
+[tool.setuptools.dynamic]
+version = {attr = "xcube_clms.__version__"}
+
```

Comment:
add 

```
[tool.setuptools.packages.find]
exclude = [
  "test*",
  "doc*"
]
```
This will exclude the test and future doc folder from being installed, if I remember correctly.